src/element/shape.js

```diff
@@ -594,43 +596,41 @@ export class ShapeElement extends Element {
   }
 
   drawShapePath(frame, shapePath, index) {
+    const beginProcess = pixiVersionHelper.select(() => {} /* for v4 API */, () => { this.beginHole(); } /* for v5 API */);
+    const endProcess = pixiVersionHelper.select(() => { this.addHole(); } /* for v4 API */, () => { this.endHole(); } /* for v5 API */);
     if (shapePath.path.hasAnimatedPath) {
       this.isClosed = shapePath.isClosed;
       const paths = shapePath.path.paths;
       if (frame < paths[0].startFrame) {
+        if (index !== 0) { beginProcess(); }
```

Comment:
I think need  `this.graphicsData.length > 1` expression for v4 .
So, `endProcess` will need guard statement like the following.

```javascript
const endProcess = pixiVersionHelper.select(() => {
    if (this.graphicsData.length <= 1) { return; }
    this.addHole();
}, () => { this.endHole(); });
```

Reply:
fixed!